services/app/src/components/header/CopyIcon.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from "prop-types";
+import CopyLink from "./headerComponents/CopyLink.svg";
+import '../../static/style/header.css';
+
+const CopyIcon = ({ roomUrl }) => {
+    const [shouldRipple, setShouldRipple] = useState(false);
+    const [clickCoords, setClickCoords] = useState({ x: -1, y: -1 })
+
+    useEffect(() => {
+        if((clickCoords.x !== -1) && (clickCoords.y !== -1)) {
+            setShouldRipple(true);
+            setTimeout(() => setShouldRipple(false), 1000);
+        }
+    }, [clickCoords]);
+
+    useEffect(() => {
+        if(!shouldRipple) setClickCoords({ x: -1, y: -1 })
+    }, [shouldRipple]);
+
+    const handleClick = (e) => {
+        setClickCoords({
```

Comment:
If possible, the UI logic (ripple effect) should be separated from the business logic (copying link into the clipboard). That way the components would be more reusable. The ripple definitelly could be reused elsewhere.